CHANGELOG.md

```diff
@@ -2,3 +2,6 @@
 ==========
 proto-route uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
+
+[PVP Versioning]: https://pvp.haskell.org/
+[on GitHub]: https://github.com/Holmusk/proto-route/blob/master/CHANGELOG.md
```

Comment:
Unfortunately, this won't work 😞 The initial content of `proto-route` is the following:

* https://raw.githubusercontent.com/Holmusk/proto-route/master/CHANGELOG.md